fala/templates/adviser/search.html

```diff
@@ -18,7 +18,7 @@ <h1 class="govuk-heading-xl"> Find a legal aid adviser or family mediator </h1>
   {% block search_form %}
       <div class="govuk-grid-column-full">
         <div class="laa-fala__grey-box">
-          <form action="" novalidate>
+          <form action="search" novalidate>
```

Comment:
Has this leaked out of the feature flag again? Or is this existing template name now used only by the flagged version, and the template that was here before is now called old_search?

Reply:
This is now the new search, so the flag is already on for this one